README.md

```diff
@@ -25,16 +25,18 @@ Compilation of all the different mods built to work with Mine Little Pony or tha
 
 ### By other devs
 1. ~[Farmer's Delight](https://modrinth.com/mod/farmers-delight-fabric)~ (only 1.20.1)
-2. ~[Seasons Mod](https://modrinth.com/mod/fabric-seasons) + [Seasons Hud](https://modrinth.com/mod/seasonhud-fabric)~ (only 1.20.1)
-3. [Falling Leaves](https://modrinth.com/mod/fallingleaves)
-4. [Missing Wilds](https://modrinth.com/mod/missing-wilds)
-5. [Apple Skin](https://modrinth.com/mod/appleskin)
-6. [Trinkets](https://modrinth.com/mod/trinkets)
-7. [Continuity](https://modrinth.com/mod/continuity)
-8. [Enchantment Descriptions](https://modrinth.com/mod/enchantment-descriptions)
-9. [Zoomify](https://modrinth.com/mod/zoomify)
-10. ~[Sound Physics Remastered](https://modrinth.com/mod/sound-physics-remastered)~ (only 1.20.1)
-11. ~[Dynamic Lights](https://modrinth.com/mod/lambdynamiclights)~ (only 1.20.1)
+2. ~[Seasons Mod](https://modrinth.com/mod/fabric-seasons)~ (only 1.20.1)
+3. ~[Serene Seasons](https://modrinth.com/mod/serene-seasons)~ (only 1.20.4) 
+4. [Seasons Hud](https://modrinth.com/mod/seasonhud-fabric)
+5. [Falling Leaves](https://modrinth.com/mod/fallingleaves)
+6. ~[Missing Wilds](https://modrinth.com/mod/missing-wilds)~ (only 1.20.1)
```

Comment:
Missing Wilds has a 1.20.2 build available. Only in beta though

https://modrinth.com/mod/missing-wilds/version/v1.2.0-beta.7+1.20.1-quilt

Reply:
I dropped 1.20.2 as it is pretty much unused by most people and is not getting updates anymore from most mods